packages/adapter-ccxt/src/types/ConfigCodecs.ts

```diff
@@ -81,10 +81,35 @@ export const ExchangeSwap = t.type({
    config: ExchangeSwapConfigCodec,
 });
 
-export const CCXTStepConfig = t.union([ExchangeWithdrawCrypto, ExchangeSwap]);
+// ---- Exchange.DepositCrypto ---- //
+
+/**
+ * Codec for the Exchange.ExchangeDepositCrypto configuration.
+ */
+export const ExchangeDepositCryptoConfigCodec = t.type({
+   exchange: t.string, // Identifier for the exchange
+   asset: t.string, // Asset to deposit e.g. "USDC"
+   toChain: t.string, // Chain identifier for destination chain
+   amount: t.number, // Amount to deposit TODO: This could be a value from the output from prev step
```

Comment:
Yeah it should be there and based on the conversation today with potential users it's likely this will not be implemented anytime soon. I will create an issue that covers the intention of this change and remove.

Reply:
It still needs to be done but have created an issue and removed the todo:
https://github.com/mento-protocol/mate/issues/24